pandas/tests/groupby/test_groupby.py

```diff
@@ -713,11 +713,8 @@ def test_ops_not_as_index(reduction_func):
     # GH 10355, 21090
     # Using as_index=False should not modify grouped column
 
-    if reduction_func in ("corrwith",):
-        pytest.skip("Test not applicable")
-
-    if reduction_func in ("nth", "ngroup"):
-        pytest.skip("Skip until behavior is determined (GH #5755)")
```

Comment:
is the GH#5755 reference no longer relevant?

Reply:
It's a closed issue, but I'll add back the reference.